UIDokan/Sources/UIDokan/ReusableViews/OTCTextField.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  OTCTextField.swift
+//  
+//
+//  Created by Ahmed Nagy on 08/07/2022.
+//
+
+import UIKit
+
+class OTCTextField: UITextField {
```

Comment:
Please use decorator pattern to separate:
- stored properties
- Public handlers
- private handlers
- Configurations